From b3a43a810c965033ce65da1efbdca934e0c8603d Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 7 Jan 2015 18:09:31 +0800
Subject: [PATCH] MLK-10244-2 imx-sdma: add new mcu_2_qspi script for i.mx7

Add new ram script for i.mx7, extend the struct.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit f4886236367ef357268e1c5d5174d012399d2764)
---
 include/linux/platform_data/dma-imx-sdma.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/platform_data/dma-imx-sdma.h b/include/linux/platform_data/dma-imx-sdma.h
index e2dcb3c..5bb0a11 100644
--- a/include/linux/platform_data/dma-imx-sdma.h
+++ b/include/linux/platform_data/dma-imx-sdma.h
@@ -52,6 +52,8 @@ struct sdma_script_start_addrs {
 	s32 zqspi_2_mcu_addr;
 	s32 mcu_2_ecspi_addr;
 	/* End of v3 array */
+	s32 mcu_2_zqspi_addr;
+	/* End of v4 array */
 };
 
 /**
-- 
1.8.0

